recipes/source.rb

```diff
@@ -50,7 +50,7 @@
 # inside of the include libraries.
 link '/usr/include/gmp.h' do
   to '/usr/include/x86_64-linux-gnu/gmp.h'
-  only_if { platform_family?('debian') && node['platform_version'].to_f >= 14.04 }
```

Comment:
Since 14.04 is EOL this can get refacetored away to just be an ubuntu check probably